### ISSUE TITLE
@ mention prioritization and/or tags

### ISSUE THREAD

**Issue by zacts** [gitterHQ/gitter#1248]
I would be interested in a feature to allow for tagged @ mentions, or equivalent functionality, to allow for prioritized mentions such as URGENT, IMPORTANT, RANDOM, WEBLINK, etc... This would allow different colors/icons or a hover quote over the channel to allow the user to see if the mention is important or not. My personal use case for this is, sometimes I like sending links to people, and I want to organize the links per nick I'm sending them to, yet I don't want to bother them with a gazillion messages that they think are urgent. Others seem to possibly find this useful too (including a nick on GitterHQ/gitter on gitter :-)): "I am also interested in a prioritization on mentions as I deal with not posting things while people are sleeping currently. Not for waking them up but so they don't think something is urgent when they check their phone in the morning or middle of the night." If I need to further explain, just let me know.

**Comment by MadLittleMods** [comment]
:+1:

>  I am also interested in a prioritization on mentions as I deal with not posting things while people are sleeping. Not for waking them up but so they don't think something is urgent when they check their phone in the morning or middle of the night.
>
> *source MadLittleMods in gitterHQ/gitter :point_up: [May 9, 2016 7:01 AM](https://gitter.im/gitterHQ/gitter?at=57307c05c2a86dcf79197731)*

**Comment by zacts** [comment]
(Just Curious) but has there been any progress on this issue? :-)

**Comment by MadLittleMods** [comment]
@zacts Nope. Progress would would be indicated by the `in progress`, `completed`, `develop`, `staging` labels or simply closing the issue unless we totally forgot to update the issue.

It isn't on our immediate roadmap so there probable won't be progress for a while unless this issue gets a lot of attention and `:+1:`'s

**Comment by MadLittleMods** [comment]
Would be interesting to distinguish a mention talking **about** you vs talking **at** you. This might be distinguished by the mention being at the front of the message but that doesn't cover every situation. Mentions talking **at** you would be prioritized higher.

Talking **at** you:
```
@MadLittleMods Start a release
@MadLittleMods When will this be ready to go?
```

Talking **about** you:
```
#xxx will be addressed when @MadLittleMods gets back tomorrow
Create an issue and assign it to @MadLittleMods
@MadLittleMods and I are going to chat later
```
